sdk/tests/py/lint/lint.py

```diff
@@ -146,10 +146,13 @@ def check_regexp_line(path, f):
 
     applicable_regexps = [regexp for regexp in regexps if regexp.applies(path)]
 
-    for i, line in enumerate(f):
-        for regexp in applicable_regexps:
-            if regexp.search(line):
-                errors.append((regexp.error, "%s line %i" % (path, i+1), i+1))
+    try:
+        for i, line in enumerate(f):
+            for regexp in applicable_regexps:
+                if regexp.search(line):
+                    errors.append((regexp.error, "%s line %i" % (path, i+1), i+1))
+    except UnicodeDecodeError as e:
+        return [("CONTAINS UNICODE", "File %s contains Unicode characters" % path, None)]
```

Comment:
Wouldn't UnicodeDecodeError mean it contains /invalid/ unicode, not just unicode?

Reply:
It seems these files are encoded with something called "extended ASCII" and not UTF-8, which Python doesn't recognize. I'm hesitant to touch their contents less I accidentally negate the tests.

Reply:
Alright! Can we call this CONTAINS NON-UNICODE then?

Reply:
or INVALID UNICODE would also work for me.

Reply:
Changed to INVALID UNICODE. Thanks.